server/src/concepts/user/repositories/commands.js

```diff
@@ -0,0 +1,38 @@
+import User from "../model/User";
+
+const createUser = (data) => {
+  const newUser = new User({
+    ...data,
+  });
+
+  try {
+    await newUser.save();
+    return newUser;
+  } catch (err) {
+    if (error.name === "MongoServerError" && error.code === 11000) {
+      throw new Error("Email must be unique");
+    } else {
+      next();
+    }
+  }
+};
+
+const updateUser = () => {
+  await User.updateOne({ name: "super_name" }, { fieldToUpdate: "new_value" });
+};
+
+const deleteUser = () => {
+  await User.deleteOne({ _id: "ObjectId" });
```

Comment:
```suggestion
  await User.deleteOne({ _id: id });
```

Please no hardcode this, provide id as parameter into this function and then use it